### PR TITLE
Enthought full stack web dev job description

### PR DESCRIPTION
**Pull request by dpinte**

### CODE REVIEW

File: jobs/enthought_web-developer.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Full Stack Web Developer 
+company: Enthought
+url: http://www.enthought.com/company/careers/
+location: Cambridge, UK
+contract: permanent
+contact:
+    name: Jill Cowan
+    email: jobs@enthought.com
+    phone:
```
Comment (salimfadhley) [inline jobs/enthought_web-developer.html]: Consider adding a phone number, or alternatively just leave this line out altogether.

File: jobs/enthought_web-developer.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+This is an opportunity to be part of a small team at Enthought to build a growing set of web services, for both internal use and customer-facing SaaS. We are looking for experienced web developers to expand the current team in our Cambridge, UK office.
+
+Responsibilities will include moving our service toward Continuous Deployment, growing the feature set of our SaaS offering, and consolidating our web stack.
+
+Our current stack uses a mix of AWS, Ansible, Nginx, Python (Flask and Django), Docker and RabbitMQ
+
+# Desired Skills and Capabilities
+
+  * 3+ years of experience with web development
```
Comment (salimfadhley) [inline jobs/enthought_web-developer.html]: Generally, we advise against job descriptions which specify years of experience. Some people learn faster than others, so it's almost never helpful and might dissuade a few fast learners!

File: jobs/enthought_web-developer.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    name: Jill Cowan
+    email: jobs@enthought.com
+    phone: 
+created: !!timestamp '2017-02-06'
+tags:
+  - uk
+  - python
+  - flask
+  - django
+---
```
Comment (salimfadhley) [inline jobs/enthought_web-developer.html]: You can also add the javascript frameworks you use as tags.

File: jobs/enthought_web-developer.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+title: Full Stack Web Developer 
+company: Enthought
+url: http://www.enthought.com/company/careers/
+location: Cambridge, UK
```
Comment (salimfadhley) [inline jobs/enthought_web-developer.html]: You can give a specific address (even with a postcode if you like) - that will allow candidates to find you on our map.

File: jobs/enthought_web-developer.html (hunk 1 above)
Comment (salimfadhley) [inline jobs/enthought_web-developer.html]: Consider adding a phone number, or alternatively just leave this line out altogether.

File: jobs/enthought_web-developer.html (hunk 2 above)
Comment (salimfadhley) [inline jobs/enthought_web-developer.html]: Generally, we advise against job descriptions which specify years of experience. Some people learn faster than others, so it's almost never helpful and might dissuade a few young quick learners! You could say "previous experience" or "demonstrable experience" as a web developer.

File: jobs/enthought_web-developer.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+Our current stack uses a mix of AWS, Ansible, Nginx, Python (Flask and Django), Docker and RabbitMQ
+
+# Desired Skills and Capabilities
+
+  * 3+ years of experience with web development
+  * History of writing well-tested code and developing through regular code review
+  * Familiarity with Flask and/or Django, SQLAlchemy (or Django ORM)
+  * Experience with one of: Ember.js, Angular.js, or React.js
+  * Ability to work independently
```
Comment (salimfadhley) [inline jobs/enthought_web-developer.html]: Candidates might get the wrong idea about this - if it's an office based job is the culture more inclined towards independent work or collaborative work.

File: jobs/enthought_web-developer.html (hunk 4 above)
Comment (dpinte) [inline jobs/enthought_web-developer.html]: @salimfadhley thanks for the review! I'll push some updates in a separate PR as this one is already merged.

File: jobs/enthought_web-developer.html (hunk 2 above)
Comment (dpinte) [inline jobs/enthought_web-developer.html]: While I tend to agree with what you suggest, in this particular case, experience requires time and exposure to a sufficiently large set of problems. Specifying an arbitrary number of years of experience is not a great way to measure that but it's an indicator.

File: jobs/enthought_web-developer.html (hunk 5 above)
Comment (dpinte) [inline jobs/enthought_web-developer.html]: I've removed this line

**Pull request merged**